zhaquirks/mli/tintE14rgbcct.py

```diff
@@ -0,0 +1,115 @@
+"""Tint E14 RGB CCT."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    GreenPowerProxy,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+MANUFACTURER_SPECIFIC_CLUSTER_ID = 0x100F  # decimal = 4111
```

Comment:
Thanks for the PR! Everything generally looks good, but:

The reason the tests are failing is that manufacturer specific clusters should only exists between ``0xFC00`` and ``0xFFFF`` (``64512`` to ``65535``).

``0x100F`` is well below that and doesn't match any existing Zigbee clusters (as far as I know).

To "fix" this, a new cluster needs to be created. Something like the following could work:
```python
class TintManufacturerSpecificCluster(CustomCluster):
    """Tint manufacturer specific cluster."""
    cluster_id = 0x1000  # not inside the manufacturer specific cluster range
```

For ``INPUT_CLUSTERS``, ``TintManufacturerSpecificCluster.cluster_id`` can still be used, but for the ``OUTPUT_CLUSTERS``, just ``TintManufacturerSpecificCluster`` needs to be used.
This *should* resolve the tests failing.

Reply:
Thanks for the tips. I'll make the changes and commit them after I test on my instance.